app/assets/javascripts/views/story_view.js

```diff
@@ -87,7 +88,9 @@ module.exports = FormView.extend({
     "click .cancel": "cancelEdit",
     "click .transition": "transition",
     "click .state-actions .estimate": "estimate",
-    "change select.story_type": "disableEstimate",
+    "change select.story_type": function(e) {
```

Comment:
Whar do you think about creating a new function and use its name here instead of using a function directly to match the standard we're following in this file?